JavaAssignments/src/shraddha/PrimeNumbersRange.java

```diff
@@ -0,0 +1,32 @@
+//-------------------------------------------------------------
+//Assignment: Sum of prime numbers in a given range of numbers.
+//-------------------------------------------------------------
+
+package shraddha;
+
+public class PrimeNumbersRange {
+
+	void PrimeNumbers(int number1, int number2) {
+		int prime_sum = 0;
+
+		for (int i = number1; i <= number2; i++) {
+			for (int j = number1; j <= number2; j++) {
```

Comment:
Please change code as per our discussion.

Reply:
Please review the changes or updated program logic